wallet/core/src/runtime/account/kind.rs

```diff
@@ -21,6 +21,32 @@ u8_try_from! {
     }
 }
 
+impl AccountKind {
+    pub fn is_legacy(&self) -> bool {
```

Comment:
I think deriving  eq+partial eq is much easier to implement and use than these helper functions

Reply:
changes: https://github.com/kaspanet/rusty-kaspa/pull/312/commits/c56940a79f0a2627b55a05e1076f9990900cc9f3

kept this one though due to it being a complex type: https://github.com/aspectron/rusty-kaspa/blob/legacy/wallet/core/src/storage/account.rs#L154